src/sentry/api/endpoints/organization_member/requests/invite/index.py

```diff
@@ -75,16 +74,14 @@ def post(self, request: Request, organization) -> Response:
 
         result = serializer.validated_data
 
-        rpc_org_member = organization_service.add_organization_member(
-            organization_id=organization.id,
-            default_org_role=organization.default_role,
-            email=result["email"],
-            role=result["role"],
-            inviter_id=request.user.id,
-            invite_status=InviteStatus.REQUESTED_TO_BE_INVITED.value,
-        )
         with transaction.atomic():
-            om = OrganizationMember.objects.get(id=rpc_org_member.id)
+            om = OrganizationMember.objects.create(
```

Comment:
maybe i'm missing something, but how does the user accepting the invite get directed to the correct silo without a org member mapping?

Reply:
We send out emails (signals attached to org member creation), which contain links that point to AuthLogin (control silo) containing enough information to route (based on OrganizationMapping which will already exist) the RPC calls that complete organization invites.

Don't forget the difference between `OrganizationMapping` (required for RPC routing) and `OrganizationMemberMapping`

Reply:
This endpoint is simply the Region API that creates invites, which triggers emails.